public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -176,6 +176,32 @@ export default class InfluxDatasource extends DataSourceApi<InfluxQuery, InfluxO
     return false;
   }
 
+  interpolateVariablesInQueries(queries: InfluxQuery[]): InfluxQuery[] {
+    let expandedQueries = queries;
+    if (queries && queries.length > 0) {
```

Comment:
if no queries return early [] (reduces nesting in rest of function).

Reply:
✅